speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/SQLDatabaseStorage.cpp

```diff
@@ -5,6 +5,8 @@
 #include "ConfigEnums.h"
 #include <config.h>
 
+#define META_BUFFER_FILENAME "sda_blackbox_internal_metadata_buffer.bin"
```

Comment:
Move this to `ConfigEnums.h`, this is where I defined all the other buffer files as well, this also allows usage in multiple files.